packages/fast-layouts-react/src/canvas/index.ts

```diff
@@ -1,6 +1,27 @@
+import * as React from "react";
+import manageJss, { ManagedJSSProps } from "@microsoft/fast-jss-manager-react";
+import {
+    Canvas as BaseCanvas,
+    CanvasClassNamesContract,
+    CanvasHandledProps as BaseCanvasHandledProps,
+    CanvasManagedClasses,
+    CanvasProps as BaseCanvasProps,
+    canvasStyleSheet,
+    CanvasUnhandledProps
+} from "./canvas";
+import { Subtract } from "utility-types";
 
-import Canvas from "./canvas";
+/* tslint:disable-next-line:typedef */
```

Comment:
Hey, I'm still a little unclear on what this is doing and when it needs to be done. Do you mind explaining it (possibly again - sorry) thanks

Reply:
It's to keep typescript from getting confused, though it may not actually be necessary with the JSSManager refactor. Previously, typescript would get confused when we would return from a function a class defined *in* that function. Now that we're not defining the class inside of a function though, this may not be necessary - I'll investigate separately.